ui/src/elements/sidebar/ngm-menu-item.ts

```diff
@@ -0,0 +1,137 @@
+import {customElement, property} from 'lit/decorators.js';
+import {LitElementI18n} from '../../i18n.js';
+import {css, html} from 'lit';
+import {classMap} from 'lit/directives/class-map.js';
+import {IconKey} from '../../icons/icons';
+import i18next from 'i18next';
+import '../shared/ngm-icon';
+
+
+@customElement('ngm-menu-item')
+export class MenuItem extends LitElementI18n {
+  @property({type: String})
+  accessor title: string = '';
+  @property()
+  accessor icon: IconKey = 'config';
+  @property({type: Boolean})
+  accessor isActive: boolean = false;
+  @property({type: Boolean})
+  accessor isMobile: boolean = false;
+
+  constructor() {
+    super();
+  }
+
+  static readonly styles = css`
+
+    :host {
+      position: relative;
+      width: 68px;
+      height: 58px;
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      color: var(--color-main);
+      padding-inline: 5px;
+      z-index: 10;
+    }
+
+    :host > .container,
+    :host > .container:active {
+      position: relative;
+      text-decoration: none;
+      width: 100%;
+      height: 100%;
+      cursor: pointer;
+    }
+
+    .container {

```

Comment:
Why are some rules for `.container` in this block, and some outside of it?
I would unify these to either have every rule in its own block, or all of them nested, unless there's a specific reason not to.